src/en/rinkocomics/src/eu/kanade/tachiyomi/extension/en/rinkocomics/RinkoComics.kt

```diff
@@ -0,0 +1,231 @@
+package eu.kanade.tachiyomi.extension.en.rinkocomics
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.POST
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import keiyoushi.utils.tryParse
+import okhttp3.FormBody
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import rx.Observable
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class RinkoComics : ParsedHttpSource() {
+
+    override val name = "Rinko Comics"
+    override val baseUrl = "https://rinkocomics.com"
+    override val lang = "en"
+    override val supportsLatest = true
+
+    private val dateFormatFull = SimpleDateFormat("MMMM d, yyyy", Locale.US)
+    private val dateFormatShort = SimpleDateFormat("MMM d, yyyy", Locale.US)
+
+    // =========================================================================
+    //  Popular (Used for listing AND Search)
+    // =========================================================================
+
+    override fun popularMangaRequest(page: Int): Request {
+        val url = if (page == 1) "$baseUrl/comic/" else "$baseUrl/comic/page/$page/"
+        return GET(url, headers)
+    }
+
+    override fun popularMangaSelector() = "article.ac-card"
+
+    override fun popularMangaFromElement(element: Element) = SManga.create().apply {
+        val anchor = element.selectFirst("h2.ac-title a") ?: element.selectFirst("a.ac-thumb")!!
+
+        setUrlWithoutDomain(anchor.attr("href"))
```

Comment:
```suggestion
        setUrlWithoutDomain(anchor.absUrl("href"))
```
(you can use the `abs:` prefix if you prefer)